### DISCUSSION TITLE
Show OpenSkill values => detailed in settings

### DISCUSSION THREAD

**Discussion by icf20** [beyond-all-reason/Beyond-All-Reason#2551] [Q&A] [answered]
the value show like ( =1,23)

what this really means ? and why is different from OS?

**Comment by L-e-x-o-n** [comment, marked as answer]
That value represents uncertainty, check this for a more detailed explanation https://www.beyondallreason.info/guide/rating-and-lobby-balance#uncertainty-sigma
